modules.d/99fs-lib/module-setup.sh

```diff
@@ -59,7 +59,7 @@ install() {
             xfs_db xfs_check xfs_repair xfs_metadump
             e2fsck jfs_fsck btrfsck
         )
-        if [[ $hostonly ]]; then
+        if [[ $hostonly_mode == "strict" ]]; then
```

Comment:
This is good, I think line 39 in `installkernel` also should change to a `hostonly_mode == "strict"` check. (Wish Github would let me comment on unchanged lines...)